vowpalwabbit/global_data.h

```diff
@@ -466,7 +481,7 @@ struct vw
   bool normalized_updates;  // Should every feature be normalized
   bool invariant_updates;   // Should we use importance aware/safe updates
   uint64_t random_seed;
```

Comment:
>random_seed [](start = 11, length = 11)

Since this is now encapsulated in random_state can we remove random_seed?

Reply:
random_seed is actually used in cb_sample.cc, lrq.cc, and nn.cc

Theoretically random_seed should be the same value that was passed in via the command line options so we can check for it there.

I guess the 2 question are:
- Which place should be the canonical location of this data?
- I don't see it used anywhere outside of reductions, should it be exposed beyond them?